src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -42,6 +42,20 @@
     //  "AllowedFileExtensions": [".jpg",".jpeg",".png",".gif",".ico",".svg",".pdf",".doc",".docx",".ppt",".pptx",".pps",".ppsx",".odt",".xls",".xlsx",".psd",".mp3",".m4a",".ogg",".wav",".mp4",".m4v",".mov",".wmv",".avi",".mpg",".ogv",".3gp"],
     //  "ContentSecurityPolicy": "default-src 'self'; style-src 'unsafe-inline'"
     //}
+    // See https://docs.orchardcore.net/en/latest/docs/reference/modules/Media.Amazon/#configuration to configure media storage in Amazon S3 Storage.
+    //"OrchardCore_Media_Amazon_S3": {
+    //  "BucketName": "",
+    // Credentials section needed only if Orchard will be hosted not in the AWS Cloud
+    // since in the AWS you don't need to store this info in the appsettings file, you 
+    // just need to set BucketName and BasePath.
```

Comment:
```suggestion
    // Credentials section needed only if Orchard will be hosted not in the AWS Cloud
    // since in AWS you don't need to store this info in the appsettings file, you just
    // need to set BucketName and BasePath.
```